integrationTests/tests/authHelper.test.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+"use strict";
+import chai, { expect } from "chai";
+import { stripBearer } from "@commerce-apps/core";
+
+describe("TS: Auth Helper Strip Bearer", () => {
```

Comment:
Is there any better way to differentiate JS/TS tests? (There might not be...)

Reply:
Open to suggestions.